src/app/app-routing.module.ts

```diff
@@ -4,12 +4,14 @@ import { Component, NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 import { DashboardComponent } from './dashboard/dashboard.component'
 import { ProfileComponent } from './profile/profile.component'
+import { SignInComponent } from './navigation/sign-in/sign-in.component';
 
 const routes: Routes = [
   {path: 'dashboard', component: DashboardComponent},
   {path: 'profile', component: ProfileComponent},
   {path: 'enterbets', component: EnterBetsComponent},
   {path: 'about', component: AboutComponent},
+  {path: 'signin',component: SignInComponent},
```

Comment:
So this is stupid and petty but good practice to put a space after the , like the above line